### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by svr333** [AudioBand/AudioBand#152]
TL;DR My focus right now is elsewhere, I will still accept Pull Requests and fix critical bugs, but there will no longer be any feature development from me. Anyone is free to fork this project, or even contribute to this repository. I will give permissions if anyone wishes to actively develop/take over this project.

Hello everyone

For the past few years I've been maintaining this project, this started out as a hobby.
As you may have noticed, there has only really been 2 updates since late 2022.

Without going into too much details, life happens and AudioBand never really made any money to live off of, though special thanks to those who did donate (TechSupport, Peter, MartialMasterWu, zevisert and probably more).
Financially I am doing okay but due to a lack of time and consequently motivation, windows 11 trying it's best to get rid of toolbar behavior and no other reason I simply don't feel like working more on this project.
This paired with me not really using my own app anymore, and me considering more and more to switch to Linux entirely, this is the end for me.

That said, I am still active on GitHub and will see any notifications regarding new issues and pull requests, so if anyone wishes to continue this project, you can do so on this organization (and reach the current install-base with our built-in updater).
If someone wishes to continue this project or take over, they can always reach me in the Discord or right here on GitHub (though I'd recommend making a few PR's first so I don't give permission to someone with bad intentions)

Thank you for all the kind words, the code contributions and the donations and a shoutout to @dsafa for creating this project in the first place!

**Comment by uzi0** [comment]
Somehow _just_ discovering AudioBand, despite being an avid user of toolbars via StartAllBack -- just wanted to say this is a brilliant project, and the love and care that's gone into it is evident :)
 
The status update is also much appreciated -- best of luck with your other endeavors 🫡

**Comment by raveren** [comment]
It was fun while it lasted, good luck and thanks for the cool project!
